sim/hunter/raptor_strike.go

```diff
@@ -114,6 +114,7 @@ func (hunter *Hunter) newRaptorStrikeHitSpell(rank int, isMH bool) *core.Spell {
 	}
 
 	return hunter.RegisterSpell(core.SpellConfig{
+		SpellCode:   SpellCode_HunterRaptorStrike,
```

Comment:
I think you probably also want to apply this to the queued spell below

Reply:
Actually you may want to do what I did in other sims and separate out spell codes for the base spell, the MH damage, the OH damage, and the queued version
```
SpellCode_HunterRaptorStrike
SpellCode_HunterRaptorStrikeMH
SpellCode_HunterRaptorStrikeOH
SpellCode_HunterRaptorStrikeQueued
```

Reply:
I kinda knew this might be commented, but is it necessary when that one line currently increases damage for all RS versions?

Added it to the queued for added security. But is the MH OH split needed?

Reply:
I did extensively test it btw

Reply:
Oh it really does it for all of them? Ok I didn't realize that

Reply:
Yeh no clue what magic causes this, but the math maths in tests